backend/src/libs/packages/mailer/mailer.ts

```diff
@@ -0,0 +1,11 @@
+import { config } from '~/libs/packages/config/config.js';
+import { logger } from '~/libs/packages/logger/logger.js';
+
+import { Mailer } from './mailer.package.js';
+
+const mailer = new Mailer({ config, logger });
+
+await mailer.connect();
```

Comment:
Not sure that we want to drop our server if there is something with email service. It's not the same as a database - an application can work even without it.